docs/concepts/serving/executor/add-endpoints.md

```diff
@@ -187,6 +187,70 @@ with Deployment(uses=MyExec) as dep:
 {"__results__": {"my_executor/rep-0": {"internal_parameter": 20.0}}}
 ```
   
+
+## Streaming endpoints
```

Comment:
You have to say that this is currently only available in HTTP protoocl and Deployment